sw/airborne/modules/relativelocalizationfilter/relativelocalizationfilter.c

```diff
@@ -0,0 +1,203 @@
+/*
+ * Copyright (C) Mario Coppola
+ *
+ * This file is part of paparazzi
+ *
+ * paparazzi is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2, or (at your option)
+ * any later version.
+ *
+ * paparazzi is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with paparazzi; see the file COPYING.  If not, see
+ * <http://www.gnu.org/licenses/>.
+ */
+/**
+ * @file "modules/relativelocalizationfilter/relativelocalizationfilter.c"
+ * @author Mario Coppola
+ * Relative Localization Filter for collision avoidance between drones
+ */
+
+#include "relativelocalizationfilter.h"
+#include "subsystems/datalink/telemetry.h"
+#include "modules/multi/traffic_info.h"
+#include "modules/stdma/stdma.h"
+#include "subsystems/datalink/bluegiga.h"
+
+#include "pprzlink/pprz_transport.h"
+#include "subsystems/datalink/datalink.h"
+#include "subsystems/datalink/downlink.h"
+
+#ifndef NUAVS
```

Comment:
If this is a setting, you should add it to the description of the module. Also, try to make the setting names more module specific.